src/ansys/aedt/core/workflows/hfss3dlayout/generate_arbitrary_wave_ports.py

```diff
@@ -259,24 +258,13 @@ def main(extension_args):
     hfss.solution_type = "Modal"
 
     # Deleting dielectric objects
-    [obj.delete() for obj in hfss.modeler.solid_objects if obj.material_name in hfss.modeler.materials.dielectrics]
+    for obj in [
+        obj for obj in hfss.modeler.solid_objects if obj.material_name in hfss.modeler.materials.dielectrics
+    ]:  # pragma no cover
+        obj.delete()
```

Comment:
```suggestion
    for solid_obj in [
        obj for obj in hfss.modeler.solid_objects if obj.material_name in hfss.modeler.materials.dielectrics
    ]: 
        solid_obj.delete()
```
Not much but it would help with readability and might avoid confusion since `obj` is already part of the comprehension list.
Also, I would prefer to remove the no cover since I don't understand why it shouldn't be covered.
Note that there is a typo: the commentary should be # pragma: no cover, not # pragma no cover